cmd/lk/dispatch.go

```diff
@@ -0,0 +1,214 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/urfave/cli/v3"
+
+	"github.com/livekit/protocol/livekit"
+	"github.com/livekit/protocol/utils"
+	lksdk "github.com/livekit/server-sdk-go/v2"
+)
+
+var (
+	DispatchCommands = []*cli.Command{
+		{
+			Name:     "dispatch",
+			Usage:    "Create, list, and delete Agent Dispatches",
```

Comment:
:nit: can all verbs have the same casing? Maybe make it `create`? feels better than making it `List` and `Delete`.

Reply:
`get` is not included in the usage here, but is there below.

Reply:
oh `get` is a specialised `list`

Reply:
yes.. i'll fix the casing.. good eye!

Reply:
I just lowercased `agent dispatches` to be consistent.. Create should be uppercased since we have been using UC for first letter of the sentence.